wordfreq/__init__.py

```diff
@@ -181,7 +186,18 @@ def _word_frequency(word, lang, wordlist, minimum):
             return minimum
         one_over_result += 1.0 / freqs[token]
 
-    return max(1.0 / one_over_result, minimum)
+    freq = 1.0 / one_over_result
+
+    if lang in SPACELESS_LANGUAGES:
+        # Divide the frequency by 10 for each token boundary that was inferred.
+        # (We determined the factor of 10 empirically by looking at words in
+        # the Chinese wordlist that weren't common enough to be identified by
+        # the tokenizer. These words would get split into multiple tokens, and
+        # their inferred frequency would be on average 9.77 times higher than
+        # their actual frequency.)
+        freq /= 10 ** (len(tokens) - 1)
```

Comment:
I wonder if this 10 would be better written as a constant (just because 10 is so common as not-an-empirical-constant).  It would almost certainly be better as 10.0.